### DISCUSSION TITLE
detection issue

### DISCUSSION THREAD

**Discussion by balajeek** [blakeblackshear/frigate#1412] [General]
I setup frigate docker and i added the following in the frigate.yml

    objects:
      track:
        - person
        - dog
        - cell phone
        - car

When i check on the web ui, it detected cars within seconds, i tried to walk in front of the camera few times and no person detection.  So the issue i have is

1. cars detected and clip is stored, but all the cars are parked in front of the road/driveway, it have the same car clips every 2 min, even though its not moving.  what should i do so it just once.
2.  what the person detection not working?

**Comment by blakeblackshear** [comment]
Please open an issue and provide the requested info in the template: https://github.com/blakeblackshear/frigate/issues/new/choose
